sandbox/kernels/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Is it necessary ? If what you want to achieve is ignore everything in this folder, you should ignore `sandbox/kernels` using your local conf.

Reply:
Oh, CUDA builds output to this directory and this is why you ignore it. Forget what I said.